docs/concepts/PrivatePayment.md

```diff
@@ -0,0 +1,12 @@
+# Private Payment
+
+Manta is all about bringing privacy to Web 3.  Supporting privacy in fungible token transfer is essential. The first and most powerful primitive we are building is _Private Payment_, more specifcally, *MantaPay*, a _multi-asset decentralized anonymous payment protocol_. 
+
+The fundamental difference between Manta and other public blockchains is that the fungbile token transfer (a.k.a. payment) is shielded by zero-knowledge proofs and thus to be private. We illustrate the difference below:
```

Comment:
'other' makes it sound like manta is public; I don't think the last sentence is necessary; tweaked to be a little more concise
```
Unlike the vast majority of blockchains, fungbile token transfers (i.e. payments) on Manta are shielded by zero-knowledge proofs, and thus private.
```